http/router.go

```diff
@@ -23,7 +23,7 @@ func Router() *mux.Router {
 	r.Use(metricsMiddleware)
 
 	// last resort error handler
-	r.Use(errors.Handler())
+	r.Use(errors.HandlerWithCounter(paceHTTPPanicCounter))
```

Comment:
Can we maintain the `errors.Handler` interface for backwards compatibility. I don't think we need both forms.

Reply:
Added a modification such that the `errors.Handler()`  stays the same